modules/repository-function/main.tf

```diff
@@ -21,12 +21,21 @@ resource "google_cloudfunctions_function" "main" {
   timeout                       = var.timeout_s
   entry_point                   = var.entry_point
   ingress_settings              = var.ingress_settings
+  trigger_http                  = var.trigger_http
   vpc_connector_egress_settings = var.vpc_connector_egress_settings
   vpc_connector                 = var.vpc_connector
 
-  event_trigger {
-    event_type = var.event_trigger["event_type"]
-    resource   = var.event_trigger["resource"]
+  dynamic "event_trigger" {
+    for_each = var.trigger_http != null ? [] : [1]
+
+    content {
+      event_type = var.event_trigger["event_type"]
+      resource   = var.event_trigger["resource"]
+
+      failure_policy {
+        retry = var.event_trigger_failure_policy_retry
```

Comment:
This var is not defined

Reply:
Thanks for the feedback, I will correct this and let you know

Reply:
@bharathkkb This has been corrected, please let me know if there are any other issues